### PR TITLE
Refactor the general step which creates the NGSI object, changing "versi...

### PR DESCRIPTION
**Pull request by jcalderin**
...on" by "api"

### CODE REVIEW

File: test/acceptance/integration/ISSUE-322 Service support for registries (NGSI9)/issue-322.feature [hunk 1]
```diff
@@ -29,7 +29,7 @@ Feature: Service support for registries (NGSI9)
 
   Scenario: Subscribe different entities with the same provider and subservices but different service
     # First registration
-    Given a new NGSI version "9" petition with the service "issue_322" and the subservice "/subservice"
+    Given a new NGSI "9" api petition with the service "issue_322" and the subservice "/subservice"
```
Comment (fgalan) [inline test/acceptance/integration/ISSUE-322 Service support for registries (NGSI9)/issue-322.feature]: petition -> request ?

Probably this change should be done in all the tests (using grep to locate all ocurrences)

Comment (fgalan) [inline test/acceptance/integration/ISSUE-322 Service support for registries (NGSI9)/issue-322.feature]: NGSI "9" -> NGSI9 (the number after NGSI token and without whitespace between them)

File: test/acceptance/integration/ISSUE-322 Service support for registries (NGSI9)/issue-322.feature [hunk 2]
```diff
@@ -143,7 +143,7 @@ Feature: Service support for registries (NGSI9)
     And build the standard context registration payload with the previous data and duration "P1M"
     And a standard context registration is asked with the before information
     # Update operation
-    And a new NGSI version "10" petition with the service "issue_322" and the subservice "/subservice"
+    And a new NGSI "10" api petition with the service "issue_322" and the subservice "/subservice"
```
Comment (fgalan) [inline test/acceptance/integration/ISSUE-322 Service support for registries (NGSI9)/issue-322.feature]: NGSI "10" -> NGSI10

File: test/acceptance/integration/ISSUE-322 Service support for registries (NGSI9)/issue-322.feature (hunk 1 above)
Comment (jcalderin) [inline test/acceptance/integration/ISSUE-322 Service support for registries (NGSI9)/issue-322.feature]: Both fixed in  3fc99d6

File: test/acceptance/integration/ISSUE-322 Service support for registries (NGSI9)/issue-322.feature (hunk 2 above)
Comment (jcalderin) [inline test/acceptance/integration/ISSUE-322 Service support for registries (NGSI9)/issue-322.feature]: Fixed in  3fc99d6

**Comment by fgalan** [comment]
It seems that the merge commit at 1422ce2 bring some new files (e.g. delete.feature) and I'm not sure if this is intended of some "merging polution" issue. Let's discuss by skype/F2F.

**Comment by fgalan** [comment]
Looking to PR https://github.com/telefonicaid/fiware-orion/pull/799, it seems to be an issue related with PR merging order. Probably it would be fixed automatically if we first work and merge #799, and then update #800 with develop.

File: test/acceptance/properties.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "logs_path": "logs"
   },
   "context_broker": {
-    "host": "localhost",
+    "host": "192.168.0.103",
```
Comment (fgalan) [inline test/acceptance/properties.json]: Not sure if this was adjusted in a previous PR to be empty... Is this correct?

Comment (jcalderin) [inline test/acceptance/properties.json]: Fixed in d0f6b25

**Comment by fgalan** [comment]
LGTM

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**